.gitignore

```diff
@@ -29,3 +29,5 @@ node_modules
 
 docs/
 dist/
+doc/GREADME.md
```

Comment:
Yeah, for now bit-docs spits out a readme file of just the API section and then I'm manually copy pasting that into the actual readme (including the header / foot stuff).  When it's updated to produce the full readme this temporary file won't be needed.